django_tenants/models.py

```diff
@@ -145,6 +147,16 @@ def get_primary_domain(self):
         except get_tenant_domain_model().DoesNotExist:
             return None
 
+    def reverse(self, request, view_name):
+        """
+        Returns the URL of this tenant.
+        """
+        url = 'https://' if request.is_secure() else 'http://'
+        url += str(self.schema_name) + '.'
+        url += get_current_site(request).domain
+        url += reverse(view_name)
+        return url
```

Comment:
`''.join()` is faster then `+` which is constructing intermediate strings, that takes memory and slows down process of building url. So it would be better to have here `''.join((protocol, self.schema_name, '.', domain, reverse(view_name))`